server.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const path = require('path');
+const app = express();
+
+app.use(express.static(path.join(__dirname, 'dist')));
+
+app.get('*', (req, res) => {
```

Comment:
What is the significance of the asterisk?

Reply:
Basically lets say we have multiple pages and someone tries to access a page that doesnt exist through the url (mainly that specific page doesnt have a url), it will basically catch that and direct them to the main page.